client/my-sites/plans/index.js

```diff
@@ -38,6 +38,14 @@ export default function() {
 		);
 
 		page(
+			'/plans/compare/:product/:domain',
+			retarget,
```

Comment:
To be clear, I'm not asking for this to be done in this PR. I'll create a separate issue for it.